rmgpy/solver/liquidTest.py

```diff
@@ -40,10 +40,20 @@
 from rmgpy.thermo import ThermoData
 from rmgpy.solver.liquid import LiquidReactor
 from rmgpy.solver.base import TerminationTime
-from rmgpy.rmg.main import RMG
 from rmgpy.rmg.settings import ModelSettings, SimulatorSettings
+from rmgpy.data.rmg import RMGDatabase
+from rmgpy.rmg.main import RMG
+from rmgpy import settings
+from rmgpy.tools.loader import loadRMGPyJob
 
 ################################################################################
+class ConcentrationPrinter(object):
+    def __init__(self):
+        self.species_names = []
+        self.data = []
+
+    def update(self, subject):
+        self.data.append((subject.t , subject.coreSpeciesConcentrations))
```

Comment:
extra space here before the comma

Reply:
This got moved to listner.py, but there is still an extra space after `subject.t`

Reply:
Sorry, missed that space. Now it is gone.